src/GUI/Pages/Project/Projects.razor.css

```diff
@@ -77,17 +75,45 @@ button[id="plus"]:focus {
     background-color: #f3f3f3;
     border-radius: 5px;
 
+    cursor: pointer;
+
     margin-bottom: 1%;
 }
 
+.project:hover {
+    background-color: #e4e4e4;
+}
+
+.project-body {
+    padding: 12px;
+}
+
+p {
+    margin-bottom: 6px;
+}
+
 .overflow-auto {
     height: 69vh;
     width: 100%;
 
     padding-right: 1%;
 }
 
-.project-addition-info {
+.project-body_general-info {
+    height: 35px;
```

Comment:
you can add display: flex; justify-content: space-between; and remove position: relative; and float for h1 and p. 
it will also.